source/interprocedural_analyses/taint/taintConfiguration.ml

```diff
@@ -16,7 +16,7 @@
 
 open Core
 open Data_structures
-module Json = Yojson.Safe
+module Json = JsonParsing.JsonAst
```

Comment:
nit: keep the name `JsonAst` to avoid confusion when reading that file, since it could refer to anything between `Yojson`, `JsonParsing`, `JsonParsing.YojsonUtils`, `JsonParsing.YojsonAst`